rakelib/artifacts.rake

```diff
@@ -197,9 +205,14 @@ namespace "artifact" do
 
   desc "Build an RPM of logstash with all dependencies"
   task "rpm_oss" => ["prepare", "generate_build_metadata"] do
-    puts("[artifact:rpm] building rpm package")
-    system("./gradlew copyJdk -Pjdk_bundle_os=linux")
-    package_with_jdk("centos", "5", :oss)
+    puts("[artifact:rpm] building rpm OSS package x86_64")
+    system("./gradlew copyJdk -Pjdk_bundle_os=linux -Pjdk_arch=x86_64")
```

Comment:
Could we combine the copying and clean up of the JDK into the `package_with_jdk` method to reduce the amount of boilerplate code?